### ISSUE TITLE
Error 0xc0000142

### ISSUE THREAD

**Issue by Car0lina5** [Alex313031/Thorium-Win#24]
Hello, I was wondering if someone could help me.
I am getting the error "0xc0000142" when trying to open the Thorium executable. The mini installer also presents a problem for me, I click on it and nothing happens, not even an error appears.
I tried it in a virtual machine with windows 11 and had the same problem, I also tried it in another one with windows 7 and had a different problem (I couldn't open it).
I tested it on someone else's computer with windows 7 and Thorium worked normally. This led me to believe that there is something missing in my computer to run the browser but I have tried everything (I've been at it for about 3 days) and I couldn't make anything work.
I would be very grateful if someone could help me... maybe tell me what needs to be installed for the browser to open on my computer. Thanks.
![25 11 2022  -  23 37  -  ](https://user-images.githubusercontent.com/117472773/204069607-9be8ad71-ee51-400d-8abd-1d441229d8dc.png)

**Comment by Car0lina5** [comment]
I can open Chrominum normally, I even used Ungoogled Chromium for quite a while. I really like to test browsers and all of them have always worked very well here. When I opened the mini installer on the other computer I noticed that it creates a folder (this didn't happen on my computer) and it also shows up in the control panel (it didn't happen for me). I don't know what is missing on my computer for it to work, the windows 7 computer I tested was freshly formatted and has almost nothing on it, but it still worked normally. This might be a stupid question but does this browser need some specific Microsoft Visual C++? Or something like that? I don't know if it helps but in the event viewer I noticed that it says that the fault is in "chrome_elf.dll". But I don't know what to do. Lastly, my processor is an Intel Pentium CPU G4560 3.50Hz. My computer is not the best, but everything has always run perfectly :/

![image](https://user-images.githubusercontent.com/117472773/204102691-d0039bc8-629b-43c5-94c7-551cc2f5a757.png)
